openwisp_radius/api/serializers.py

```diff
@@ -102,15 +105,15 @@ class Meta:
         fields = '__all__'
 
 
-class RadiusBatchSerializer(serializers.ModelSerializer):
+class RadiusBatchSerializer(ValidatedModelSerializer):
+    organization = serializers.SlugRelatedField(
+        queryset=Organization.objects.all(), slug_field='slug',
+    )
     users = UserSerializer(many=True, read_only=True)
     prefix = serializers.CharField(required=False)
     csvfile = serializers.FileField(required=False)
     pdf = serializers.FileField(required=False, read_only=True)
     pdf_link = serializers.SerializerMethodField(required=False, read_only=True)
-    number_of_users = serializers.IntegerField(
-        required=False, write_only=True, min_value=1
-    )
```

Comment:
To use `ValidatedModelSerializer`, I need to remove `number_of_users`; However, I don't think `full_clean()` is required here, all the values are checked by serializer now. What do you think?